src/lib/y2partitioner/actions/controllers/md.rb

```diff
@@ -287,7 +287,9 @@ def available_disks
         # @param partition [Y2Storage::Partition]
         # @return [Boolean] true if it is valid; false otherwise.
         def valid_for_md?(partition)
-          partition.id.is?(:linux_system) && !partition.type.is?(:extended)
+          partition.id.is?(:linux_system) &&
+            !partition.type.is?(:extended) &&
+            !partition.partitionable.is?(:raid)
```

Comment:
method documentation does not mention that it filter out raid partitions. I also would like to see here in code rationale why we filter it out, as we easily forgot WHY in few months.